pkg/menu/menu_test.go

```diff
@@ -0,0 +1,55 @@
+package menu
+
+import (
+	ui "github.com/gizak/termui/v3"
+	"strconv"
+	"testing"
+)
+
+func Test_newParagraph(t *testing.T) {
+	if err := ui.Init(); err != nil {
+		t.Fatal(err)
+	}
+	defer ui.Close()
+	const testText = "newParagraph test"
+	p := newParagraph(testText, false, 0, 50, 3)
+	if testText != p.Text {
+		t.Errorf("p.Text is %s, expect %s", p.Text, testText)
```

Comment:
Nit: I think convention for errors in Go tests is something like `t.Errorf("incorrect value for p.Text. got: %v, want: %v", p.Text, testText)`